spec/support/command_line_helpers.rb

```diff
@@ -17,7 +18,7 @@ def stub_run_for(module_to_stub)
     module_to_stub.stub(:run) do |command, options|
       options ||= {}
       $commands_ran << Hashie::Mash.new(command: command, options: options)
-      command = "" # we need this due to a bug in rspec that will keep this assignment on subsequent runs of the stub
```

Comment:
I don't get the comment here, but all tests pass. So, I changed it.
Could you elaborate more on the comment a little? I'd like to understand it.